x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_embeddable_factory.ts

```diff
@@ -37,7 +46,7 @@ export class AnomalyChartsEmbeddableFactory
 
   public getDisplayName() {
     return i18n.translate('xpack.ml.components.mlAnomalyExplorerEmbeddable.displayName', {
-      defaultMessage: 'ML anomaly chart',
+      defaultMessage: 'Anomaly chart',
```

Comment:
For the "Anomaly chart" embeddable, I suggest using text like this in the tooltip: "View anomaly detection results in a chart".